sidebars.js

```diff
@@ -8,93 +8,115 @@
-                "web/concepts-key-concepts",
-                "web/concepts-layout",
-                "web/concepts-app-config",
-                "web/concepts-commands-operations",
-                "web/concepts-components-services",
-                "web/concepts-events",
                 {
                     type: "category",
-                    label: "Advanced Concepts",
+                    label: "Layout",
+                    items: [
+                        "web/configuration-layout",
+                        "web/configuration-layout-reference",
+                        "web/configuration-advanced-model-binding",
+                    ],
+                },
+                {
+                    type: "category",
+                    label: "App Config",
+                    items: [
+                        "web/configuration-app-config",
+                        "web/configuration-app-config-reference",
+                        "web/configuration-theme",
+                    ],
+                },
+                "web/configuration-commands-operations",
+                {
+                    type: "category",
+                    label: "Tutorials",
                     items: [
-                        "web/concepts-advanced-layout",
-                        "web/concepts-advanced-app-config",
+                        "web/tutorial-configure-button-click",
+                        "web/tutorial-change-default-behavior",
+                        "web/tutorial-customize-application-theme",
                     ],
                 },
             ],
         },
         {
             type: "category",
-            label: "Use Cases",
+            label: "Workflow",
+            items: [
+                "web/tutorial-run-workflow-app-config",
+                "web/tutorial-display-custom-form",
+                "web/tutorial-change-default-map-click-behavior",
+                "web/tutorial-implement-custom-activity",
+                "web/tutorial-implement-custom-form-elements",
+            ],
+        },
+        {
+            type: "category",
+            label: "Geocortex Web SDK",
             items: [
+                "web/sdk-overview",
                 {
                     type: "category",
-                    label: "Basics",
+                    label: "Components",
                     items: [
-                        "web/usecases-change-default-behavior",
-                        "web/usecases-look-and-feel",
-                        "web/usecases-internationalization",
+                        "web/sdk-components-overview",
+                        "web/sdk-create-component",
+                        "web/sdk-component-reference",
+                        "web/sdk-component-hooks",
+                        "web/sdk-ui-context",
+                        "web/sdk-components-commands-operations",
+                        "web/sdk-component-interactions",
+                        "web/sdk-service-injection",
```

Comment:
I notice that this is used in two spots within our sidebar hierarchy:

![image](https://user-images.githubusercontent.com/6355370/81587330-bd018a80-936b-11ea-9551-f0a9892aca6d.png)

I understand the thinking as it applies to both, but it looks a little strange and will most likely confuse Aloglia as it uses the active links both at the top nav and in sidebar to categorize the content.

Reply:
Yeah... It definitely will confuse Algolia, and also looks strange. I broke it up into two similiar but distinct articles to not confuse algolia and introduce weird sidebar behavior. 

I debated making the article into a snippet, as it has no headings that would limit the articles content from being shared, but it didn't seem like there was enough benefit from sharing the content.